set.mm

```diff
@@ -654308,8 +654331,8 @@ standardize vectors to a length (norm) of one, but such definitions require
     $( Since ` A =/= B ` by ~ fltne , we may assume ` A < B ` without loss of
        generality. $)
     fltnlta.1 $e |- ( ph -> A < B ) $.
-    $( ` N ` is less than ` A ` .  See ~ https://www.youtu.be/EymVXkPWxyc for
-       an outline.  (Contributed by Steven Nguyen, 24-Aug-2023.) $)
+    $( ` N ` is less than ` A ` .  See ~ https://youtu.be/EymVXkPWxyc for an
+       outline.  (Contributed by Steven Nguyen, 24-Aug-2023.) $)
```

Comment:
Link seems broken. A less cryptic comment may be:
> In a Fermat counterexample, the exponent is less than all three numbers.

(or "all three bases" or another term that you use for these)